src/features/authentication/authReducer.js

```diff
@@ -1,12 +1,13 @@
-
 import loginReducer from './login/loginReducer';
 import passwordResetReducer from './password-reset/passwordResetReducer';
 import profileReducer from './profile/profileReducer';
 import signupReducer from './signup/signupReducer';
+import localStorageUtil from '../../shared/utilities/localStorageUtil';
 
 const initialState = {
-  user: {},
+  user: localStorageUtil.getItem('ah_user') || {},
   errors: {},
+  token: '',
```

Comment:
As I explained in my previous PR, I intentionally removed this as it was preventing the logout function from working. Besides that, this is not really necessary as the whole state is already persisted in local storage.